### PR TITLE
[sc-4030] video change for player

### PR DESCRIPTION
**Pull request by StephanieZuniga**
Add support for video change for player

### CODE REVIEW

File: MUXSDKKaltura/MUXSDKKaltura.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -385,7 +385,7 @@
 				"${BUILT_PRODUCTS_DIR}/PlayKitUtils/PlayKitUtils.framework",
 				"${BUILT_PRODUCTS_DIR}/SwiftyJSON/SwiftyJSON.framework",
 				"${BUILT_PRODUCTS_DIR}/XCGLogger/XCGLogger.framework",
-				"${PODS_XCFRAMEWORKS_BUILD_DIR}/MuxCore/MuxCore.framework/MuxCore",
+				"${PODS_XCFRAMEWORKS_BUILD_DIR}/Mux-Stats-Core/MuxCore.framework/MuxCore",
```
Comment (bmulligan-kg) [inline MUXSDKKaltura/MUXSDKKaltura.xcodeproj/project.pbxproj:388]: ??

File: Mux-Stats-Kaltura.podspec [hunk 2]
```diff
@@ -26,6 +26,6 @@ Pod::Spec.new do |s|
   s.tvos.deployment_target = '9.0'
   s.tvos.vendored_frameworks = 'XCFramework/MUXSDKKaltura.xcframework'
 
-  s.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'i386' }
+  s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
```
Comment (bmulligan-kg) [inline Mux-Stats-Kaltura.podspec:29]: will this not be overwritten by the following line?

Comment (StephanieZuniga) [reply Mux-Stats-Kaltura.podspec:29]: oh I think you're right. I didn't know how this works but you're right we probably need to put all the settings in the same line. Yeah... I'll update, thanks.

Comment (StephanieZuniga) [reply Mux-Stats-Kaltura.podspec]: updated

**Review by bmulligan-kg — APPROVED** [review]

**Review by nidhik — APPROVED** [review]

**Pull request merged**